CMake/Dependencies/libwebsockets-CMakeLists.txt

```diff
@@ -20,7 +20,7 @@ endif()
 
 ExternalProject_Add(project_libwebsockets
     GIT_REPOSITORY    https://github.com/warmcat/libwebsockets.git
-    GIT_TAG           v3.2.3
+    GIT_TAG           v4.2-stable
```

Comment:
Can you please change this to the latest release that just went out and we can quickly get this merged:

https://github.com/warmcat/libwebsockets/releases/tag/v4.2.1